_includes/pbc/all/install-features/202311.0/install-the-push-notification-feature.md

```diff
@@ -159,7 +159,7 @@ $config[PushNotificationWebPushPhpConstants::VAPID_PRIVATE_KEY] = getenv('SPRYKE
 $config[PushNotificationWebPushPhpConstants::VAPID_SUJECT] = getenv('SPRYKER_PUSH_NOTIFICATION_WEB_PUSH_PHP_VAPID_SUBJECT');
 ```
 
-3. Add the VAPID keys to the needed deploy file:
+3. Add the VAPID keys to the needed local deploy file and to variables in the Parameter Store:
```

Comment:
I would separate this into 2 sections, since the person who doesn't read the text, will only use YML file update